packages/dev-frontend/src/components/Dashboard/Chart/LineChart.tsx

```diff
@@ -68,11 +68,12 @@ export const LineChart = ({ dataTitle, tooltipText }: LineChartProps): JSX.Eleme
   const [loadedChart, setLoadedChart] = useState<boolean>(false);
   const [activeLabel, setActiveLabel] = useState<string>('-');
 
-  const formatData = (data: string) => {
+  const formatData = (data: string, timestamp: number) => {
     if (dataTitle == 'LED APY') {
       const redemptionRate = Decimal.from(data).div(Decimal.from(10).pow(27));
-      const secondsPerYear = 31536000; // Approximate number of seconds in a year
-      const ratePerYear = redemptionRate.pow(secondsPerYear);
+      const tournamentEndDate = Math.floor(new Date('Jul 08 2023 19:00 GMT').getTime() / 1000);
+      const secondsUntilEnd = tournamentEndDate - timestamp; // interest rate at time of collection
+      const ratePerYear = redemptionRate.pow(secondsUntilEnd);
```

Comment:
maybe we should add a check that the current time isn't greater than the tournament end time